src/thor/multimodal.cc

```diff
@@ -740,21 +740,19 @@ bool MultiModalPathAlgorithm::ExpandFromNode(baldr::GraphReader& graphreader,
 
     // Check if lower cost path
     if (es->set() == EdgeSet::kTemporary) {
-      EdgeLabel& lab = edgelabels[es->index()];
+      auto& lab = edgelabels[es->index()];
       if (newcost.cost < lab.cost().cost) {
-        float newsortcost = lab.sortcost() - (lab.cost().cost - newcost.cost);
```

Comment:
`newcost.cost` is now equivalent to this?

Reply:
this is the same as what is changed in timedistancematrix code. In these cases there is no A* heuristsic, so the sort cost is the new cost. This basically triggers the adjacency list method to decrease the sort cost in the queue.